src/NUnitFramework/mock-assembly/mock-assembly.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
I don't think .NET Standard tests are supported. We should try to test as close to possible what we support, and we tell people to make their test assemblies target concrete platforms (which is any target except .NET Standard).

https://docs.nunit.org/articles/nunit/getting-started/dotnet-core-and-dotnet-standard.html#why-cant-my-tests-target-net-standard

Reply:
@jnm2 You would be right if this assembly actually was a test assembly. However the only test assembly is `nunit.framework.tests`, i.e. an assembly that references the `Microsot.NET.Test.Sdk` and `NUnit3TestAdapter`. That is also the only one tested in `build.cake`. All others, including the `mock-assembly` are referenced in that one.

My understanding is that the `TargetFramework` for a referenced assembly defines the API available on that platform, Only once it is executed, the `RunTimeFramework` comes into play.
There is no conditional compilation in the source code to behave differently between `netstandard2.0` and later.

If my understanding is wrong, please correct me.

Reply:
Maybe it doesn't matter, but I thought the point of mock-assembly and the testdata assembly was to mimic a real test assembly as closely as possible. `build.cake` only invokes `nunit.framework.tests`, but then `nunit.framework.tests` takes on the same role as `build.cake` as a test executor to test execution of tests.

Reply:
Ok. 
To ensure that these satellite assemblies use the same set of frameworks as the main `nunit.framework.tests`, I have added two properties to `Directory.Build.props`:
* `NUnitLibraryFrameworks`, used for libraries.
* `NUnitRuntimeFrameworks`, used for all test assemblies and nunitlite-runner.

This should get consistency and prevents diverging.
The only difference is that in build.cake the frameworks are still listed separately.
It would be good to fix #3867 so we could simply run `dotnet test` to test on all supported runtime frameworks.

Reply:
@jnm2 As you seem to be happy, can you approve please?